### ISSUE TITLE
The ITM and LM loss do not converge

### ISSUE THREAD

**Issue by chenyzh28** [salesforce/BLIP#152]
Thanks for your work! But I have met some problems when I pretrain BLIP on a chinese dataset (about 4M, coco + vg + some part of cc3m and cc12m). I found that the ITA loss can converge after 20 epochs, but the ITM and LM loss are still high. 
![企业微信截图_2718cf3e-622a-42ad-9200-d37b10f8ed0d](https://user-images.githubusercontent.com/35193979/237017620-a363cfae-c0cf-41a7-95d5-37c1f304bab3.png)
I also test the model performance on COCO validation set (chinese), and it seems normal if I only calculate the ITC similarity.
![企业微信截图_6a7dce82-3bd3-48e8-a96a-575d45aa7e32](https://user-images.githubusercontent.com/35193979/237018585-2c7fb782-8c0e-45f9-a292-23793b277f3e.png)
But when I use the ITM score, since the model does not converge on ITM and LM tasks, the evaluation performance is poor. I notice that `score_matrix_i2t[start+i,topk_idx] = score + topk_sim`, so the TR benefits from ITC task and nothing from ITM task. 
![企业微信截图_56b48076-961e-41d9-b266-315ba20958df](https://user-images.githubusercontent.com/35193979/237018921-7ea9f48f-5d31-4244-946b-2ef59d8f9b7c.png)
Do you know where the problem might be? It would be greatly appreciated if any advice is given.
When I pretrain on chinese dataset, I translate the caption to chinese, and initialize the tokenizer and text encoder with "bert-base-chinese", no other modules have been modified.
![企业微信截图_80f5bf4b-4a66-45de-bea2-7a688da969e1](https://user-images.githubusercontent.com/35193979/237020273-7810b27f-d1a9-42bd-86f5-12425bc0c195.png)

**Comment by Hoogck** [comment]
I have the same problem as you，did you solve it?

**Comment by chenyzh28** [comment]
@Hoogck Yes. Please refer to https://github.com/salesforce/LAVIS/issues/282#issuecomment-1639685298

**Comment by Hoogck** [comment]
> @Hoogck Yes. Please refer to [salesforce/LAVIS#282 (comment)](https://github.com/salesforce/LAVIS/issues/282#issuecomment-1639685298)

great ,thank you very much!
